sonar-perl-plugin/build.gradle

```diff
@@ -37,7 +37,7 @@ jar {
         attributes "Build-Jdk": System.getProperty('java.version'),
                    "Build-Time": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ"),
                    "Plugin-BuildDate": new Date().format("yyyy-MM-dd'T'HH:mm:ssZ"),
-                   'Plugin-Key': 'perl',
+                   'Plugin-Key': 'sonarperl',
```

Comment:
Key should not include `sonar` as they might consider to exclude that in their rules, see the discussion with Ann at
https://community.sonarsource.com/t/new-release-perl-plugin-0-4-0/5569/8

How about `communityperl`?